storm-core/src/jvm/org/apache/storm/topology/base/BaseWindowedBolt.java

```diff
@@ -255,6 +255,17 @@ public BaseWindowedBolt withTimestampField(String fieldName) {
     }
 
     /**
+     * Turns on or off, on a per component basis, the emission of late tuples on a stream
+     * called _late.
+     *
+     * @param flag the flag to indicate whether to emit late tuples
+     */
+    public BaseWindowedBolt withLateTupleEmission(Boolean flag) {
+        windowConfiguration.put(Config.TOPOLOGY_BOLTS_EMIT_LATE_TUPLE, flag);
```

Comment:
good to add validation to ensure this is used with timestamp field (like you had in your earlier patch).